qemu/tests/cfg/pvpanic_event_check.cfg

```diff
@@ -14,4 +14,7 @@
     i440fx:
         expected_cap = 1
     q35:
-        expected_cap = 3
+        RHEL.9:
+            expected_cap = 3
+        RHEL.10:
+            expected_cap = 7
```

Comment:
```suggestion
        expected_cap = 7
        RHEL.9:
            expected_cap = 3
```
In general, we should set a default value for it, which should be the value it has in your main test version. Then, you can specify the value of this parameter separately based on testing needs. cc @vivianQizhu for more comments on it. Thanks.

Reply:
I agree with you, @leidwang. PVPANIC_SHUTDOWN should be supported in future RHEL versions.

Reply:
Should this be determinded by qemu version instead of RHEL version?